app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,22 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(
+            self,
+            nickname: str,
+            musical_instrument: str,
+            bow_level: int
+    ) -> None:
+        super().__init__(
+            nickname=nickname,
+            musical_instrument=musical_instrument
+        )
+        self.bow_level = bow_level
+
+    def player_info(self) -> str:
+        return f"Elf ranger {self.nickname}." \
+               f" {self.nickname} has bow of the {self.bow_level} level"

```

Comment:
also please use `()` here and other places 
```suggestion
        return (f"Elf ranger {self.nickname}." 
                f" {self.nickname} has bow of the {self.bow_level} level")

```